packages/chakra-components/src/components/Election/VoteButton.tsx

```diff
@@ -67,3 +68,26 @@ export const VoteButton = (props: ButtonProps) => {
 
   return <Button {...button} />
 }
+
+export const VoteWeight = () => {
+  const { client, election, localize } = useElection()
+  const [weight, setWeight] = useState<string | null>(null)
+  const styles = useMultiStyleConfig('VoteWeight')
+
+  useEffect(() => {
```

Comment:
Please always try to add comments to effects, explaining what the effect does.

Reply:
8c4a788